Expensify/App
# Evict largest never accessed key then last accessed key when we hit localStorage limits

cc @alex-mechler @iwiznia @tgolen 

### Fixed Issues (For now...)
Fixes https://github.com/Expensify/Expensify/issues/143175

### Tests
1. Create 5 chat reports between a brand new user and 5 other users (save usernames and reportIDs)
1. Add 2000 comments to each one of those reports (**Tip:** Use `clitools.sh generator:reportcomment`)
1. This should amount to about 10mb of data which should be enough to break localStorage
1. Once you have enough large reports you should see the following `console.log` in the JS console after putting the web app into the `background` and bringing it back into `active`:
>[Ion] Max storage reached. Evicting largest key that has never been accessed and retrying. {keyForRemoval: "reportActions_200"}
5. Add a comment to each report as the other participant (not the currently logged in user) so that the reports are marked as unread and refresh after leaving each comment
6. See that it's not possible to run out of space and the active report in view never has it's report actions removed.
7. Attempt to switch to one of the chats that has been removed and verify the report actions are refetched.

### Screenshots
❌
